microbench/src/main/java/io/netty/microbench/headers/HeadersBenchmark.java

```diff
@@ -139,4 +163,546 @@ public void http2Iterate(Blackhole bh) {
             bh.consume(entry);
         }
     }
+
+    @Benchmark
+    @BenchmarkMode(Mode.AverageTime)
+    public void httpAddAllFastest(Blackhole bh) {
+        bh.consume(new DefaultHttpHeaders(false).add(httpHeaders));
```

Comment:
Happy to make the change though I agree it will have marginal impact on the actual numbers.